external-api/src/helpers/cron.ts

```diff
@@ -33,6 +34,8 @@ const onTick = async (eStoreData: Estore, category: Category) => {
       return eStoreDealDirectoryCreationJob(eStoreData);
     case ENDPOINT.FACILITY:
       return eStoreFacilityDirectoryCreationJob(eStoreData);
+    case ENDPOINT.DOCUMENT:
+      return eStoreDocumentsCreationJob(eStoreData);
```

Comment:
Not a blocker as it's already existing, the constant is `DOCUMENT`, but the function is `eStoreDocuments...`, should we rename `DOCUMENT` to `DOCUMENTS`?

Reply:
Good point, I have kept all of them singular as document can be either none, one or more than one.

```typescript
export const ENDPOINT = {
  SITE: 'sites',
  TERM: 'terms/facilities',
  BUYER: 'buyers',
  DEAL: 'deals',
  FACILITY: 'facilities',
  DOCUMENT: 'documents',
} as const;
```

Reply:
OK makes sense for now - not a blocker but I would vote for adding S to all of them to match the actual value